benchmarks/yamauchi_takei_2016_anelasticity/anelasticity_temperature.cc

```diff
@@ -0,0 +1,469 @@
+	                ( mflag && (std::abs(b-c) < TOL) ) ||
+	                ( !mflag && (std::abs(c-d) < TOL))  )
+	        {
+	            // bisection method
+	            s = (a+b)*0.5;
+	            mflag = true;
+	        }
+	        else
+	        {
+	            mflag = false;
+	        }
+	        fs = f(s);  // calculate fs
+	        d = c;      // first time d is being used (wasnt used on first iteration because mflag was set)
+	        c = b;      // set c equal to upper bound
+	        fc = fb;    // set f(c) = f(b)
+	        if ( fa * fs < 0)   // fa and fs have opposite signs
+	        {
+	            b = s;
+	            fb = fs;    // set f(b) = f(s)
+	        }
+	        else
+	        {
+	            a = s;
+	            fa = fs;    // set f(a) = f(s)
+	        }
+	        if (std::abs(fa) < std::abs(fb)) // if magnitude of fa is less than magnitude of fb
+	        {
+	            std::swap(a,b);     // swap a and b
+	            std::swap(fa,fb);   // make sure f(a) and f(b) are correct after swap
+	        }
+	    } // end for
+	    std::cout<< "The solution does not converge or iterations are not sufficient" << std::endl;
+        return 0;
+	} // end brent
+
+    template <int dim>
+    void
+    AnelasticVs2T<dim>::declare_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection ("Initial temperature model");
+      {
+        prm.declare_entry ("Remove temperature heterogeneity down to specified depth", 
+						 boost::lexical_cast<std::string>(-std::numeric_limits<double>::max()),
+                         Patterns::Double (),
+                         "This will set the heterogeneity prescribed by S20RTS or S40RTS to zero "
+                         "down to the specified depth (in meters). Note that your resolution has "
+                         "to be adequate to capture this cutoff. For example if you specify a depth "
+                         "of 660km, but your closest spherical depth layers are only at 500km and "
+                         "750km (due to a coarse resolution) it will only zero out heterogeneities "
+                         "down to 500km. Similar caution has to be taken when using adaptive meshing.");
+        prm.declare_entry ("Use Yamauchi and Takei parameterization", "true", 
+						 Patterns::Bool(),
+                         "This parameter determines whether to use the anelasticity model of "
+						 "Yamauchi & Takei, 2016 (JGR) to convert absolute Vs into temperature");
+        prm.declare_entry ("Use original density and frequency model of Yamauchi and Takei", "true",
+						 Patterns::Bool(),
+						 "Use original density and frequency model of Yamauchi and Takei where density"
+						 "has simple pressure-dependence and shear wave is period set at 100s");
+        
+		Utilities::AsciiDataBase<dim>::declare_parameters(prm,
+                         "$ASPECT_SOURCE_DIR/data/initial-temperature/ascii-data/test/",
+                         "box_2d.txt");
```

Comment:
is this a sensible default?

Reply:
I have changed this to box_2d_Vs_YT16.txt which is the 2D example Vs input used in the benchmark.